src/attribute_converter.cc

```diff
@@ -185,8 +185,9 @@ void GlobalDictionary::ShrinkToBase() {
   }
 }
 
-void AttributeConverter::Convert(const Attributes& attributes,
-                                 ::istio::mixer::v1::Attributes* pb) const {
+void AttributeConverter::Convert(
```

Comment:
Name this `Compress`?

Reply:
There are too many places called "convert".  AttributerConverter as class,   attribute_converter as fie name.  Maybe a separate PR.

Reply:
OK